.eslintignore

```diff
@@ -0,0 +1,2 @@
+test
```

Comment:
I don't think we actually want to ignore test, do we? I'd like our test code to be linted too.

Reply:
That's fair. This was earlier in the process of making these PRs when I still wasn't completely sure how to handle "exclude `test` from `tsconfig.json` but not from linting", and now I've figured it out.

I'll revisit.